src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2252,19 +2278,42 @@ export default class AircraftModel {
      * @method updateAircraftTurnPhysics
      */
     updateAircraftTurnPhysics() {
-        if (this.isOnGround() || this.heading === this.target.heading) {
+        if (this.isOnGround()) {
+            this.target.turn = null;
+
+            return;
+        }
+
+        if (!_isNil(this.targetGroundTrack)) {
+            // TODO: This isn't exactly right, as the groundspeed will be different
+            // at the targeted track than it is at the current one, though it is a fine approximation
+            const groundTrackVector = vscale(vectorize_2d(this.targetGroundTrack), this.groundSpeed);
+
+            // TODO: This is copied from this.updateGroundSpeedPhysics(). Should be moved somewhere it's used by both
+            // Calculate wind vector
+            const windIncreaseFactorPerFoot = 0.00002; // 2.00% per thousand feet
+            const { wind } = AirportController.airport_get();
+            const windTravelDirection = wind.angle + Math.PI;
+            const windTravelSpeedAtSurface = wind.speed;
+            const windTravelSpeed = windTravelSpeedAtSurface * (1 + (this.altitude * windIncreaseFactorPerFoot));
+            const windVector = vscale(vectorize_2d(windTravelDirection), windTravelSpeed);
+
+            this.targetHeading = vradial(vsub(groundTrackVector, windVector));
```

Comment:
Don't change the target heading or target ground track. Simply get the heading to use rather than changing stuff:
```
let targetHeading = this.targetHeading;

if (_isNil(targetHeading)) {
    targetHeading = this.calculateCrabHeadingForGroundTrack(this.targetGroundTrack);
}
```

Reply:
done!